tools/c7n_tencentcloud/c7n_tencentcloud/filters/metrics.py

```diff
@@ -29,7 +29,12 @@
 
 
 class MetricsFilter(Filter):
-    """MetricsFilter"""
+    """Supports metrics filters on resources.
```

Comment:
this should have an actual policy example

Reply:
done